BaaServiceKitTests/Mocks/MockAPIClient.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MockAPIClient.swift
+//  BaaServiceKitTests
+//
+//  Created by Daniel Metzing on 26.08.18.
+//  Copyright © 2018 DirtyLabs. All rights reserved.
+//
+
+import Foundation
+@testable import BaaServiceKit
+
+final class MockAPIClient: APIClient {
+
+    var result: Result<APIResponse>?
+
+    override func execute(request: BlockchainRequest, completion: @escaping (Result<APIResponse>) -> Swift.Void) {
```

Comment:
Why do you use `Swift.Void`? Why dont enough `Void`?

Reply:
Its just coming from SwiftLint, I prefer to use `Swift.Void`